frame/staking/src/pallet/mod.rs

```diff
@@ -272,6 +271,15 @@ pub mod pallet {
 		type WeightInfo: WeightInfo;
 	}
 
+	/// Maximum limit of nominations per nominator, regardless of `T::NominationsQuota`.
+	#[pallet::extra_constants]
+	impl<T: Config> Pallet<T> {
+		#[pallet::constant_name(MaxNominations)]
```

Comment:
Wasn't there supposed to be a runtime API for this as well? would be good to document in code and in general how this should be used next to the runtime API

Reply:
The idea here was to expose the `AbsoluteMaxNominations` configured, but I think this is not that useful anymore. Removing it.